rclcpp/src/rclcpp/parameter_client.cpp

```diff
@@ -228,11 +228,11 @@ AsyncParametersClient::list_parameters(
 }
 
 SyncParametersClient::SyncParametersClient(
-  rclcpp::node::Node::SharedPtr node)
+  rclcpp::node::Node::SharedPtr node,const std::string & remote_node_name)
```

Comment:
You need to have a single space after the `,`.

In the future you can run out linter tools by running the tests for `rclcpp` or by running `ament_cpplint path/to/rclcpp`, `ament_uncrustify path/to/rclcpp`, etc... manually. You have to do this in a terminal where you sourced the setup file after building.